### PR TITLE
Rust survey results (draft)

### PR DESCRIPTION
**Pull request by erlend-sh**
A few changes still pending, in particular the wordcloud graphic for "libraries you're thankful for"

I'll share access to the form on Google Docs with the other WG leads.

### CODE REVIEW

**Comment by AlexEne** [comment]
This is really good data. Thanks for all the work you have done. So happy that map has debug visualization and cargo profiles will also solve the debug config pains :)

**Comment by erlend-sh** [comment]
Alright, this is ready for final review now. I've incorporated your feedback @17cupsofcoffee , thanks!

File: content/posts/survey-01/index.md [hunk 1]
```diff
@@ -3,16 +3,19 @@ title = "Rust Game Development - Ecosystem Survey"
...
 
-## Are you using Rust for gamedev?
+# Are you using Rust for gamedev?
 ![Respondents using Rust in game development](using-rust.png)
+Among the Professionals only 1/4 said they are currently using Rust in games. That means the 200 respondents currently using Rust for gamedev are mostly hobbyists. No surprise there, as most professionals can't bear the risk of being an early adopter. Big shoutout to all the hobbyists who are pushing the status quo, and kudos to the handful of studios that are using Rust in production long before most of their industry peers have even caught the scent.
+
+---
 
-For the free-form answers that follow, a basic sentiment analysis was used to distill 400 replies into digestible form. The "III" signs underneath a heading indicate *roughly* the number of times that topic came up. Think of it as the **signal strength** of a given topic.
+_For the free-form answers that follow, a basic eyeballs&hands sentiment analysis was used to distill 400 replies into digestible form. The (#) number next to a heading indicate roughly the number of times that topic came up. Think of it as the **signal strength** of a given topic._
```
Comment (Wodann) [inline content/posts/survey-01/index.md]:
```suggestion
_For the free-form answers that follow, a basic eyeballs&hands sentiment analysis was used to distill 400 replies into digestible form. The (#) number next to a heading roughly indicates the number of times that topic came up. Think of it as the **signal strength** of a given topic._
```

File: content/posts/survey-01/index.md [hunk 2]
```diff
@@ -263,31 +241,27 @@ IIiIIIIIIIIIIIIIIIiIIIIIIII
...
-### Interoperability between libraries
-IIIII
+### Interoperability between libraries (5)
 
 ## Can you name some libraries you are thankful for or take special interest in?
-
-[graphic pending]
+Just for fun, we asked respondents to name libraries or projects they were thankful for. In the 6 months that have passed since this survey went out there's been quite a few newcomers on the rise that would probably show more prominently here if we asked again today. Rest assured there will be more surveys.
+![Shoutouts to ecosystem projects](survey-thanks.png)
+_Thanks so much to the survey respondents and all the people out there working tirelessly make Rust a first-tier language and ecosystem for game development._
```
Comment (Wodann) [inline content/posts/survey-01/index.md]:
```suggestion
_Thank you so much to the survey respondents and all the people out there working tirelessly to make Rust a first-tier language and ecosystem for game development._
```

Comment (erlend-sh) [reply content/posts/survey-01/index.md]: Has this been applied? I haven't used this GH feature before so I'm not quite sure what happened when I committed your first set of suggestions. These other suggestions are apparently "outdated" now.

Comment (Wodann) [reply content/posts/survey-01/index.md]: The second suggestion seems not to have been applied.

Comment (erlend-sh) [reply content/posts/survey-01/index.md]: Ah, I guess I should’ve opted for “add suggestion to batch” instead. Could you please resubmit the missing suggestions?

File: content/posts/survey-01/index.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+* The language has a higher upfront cognitive load than any other mainstream language I know of, mostly due to the borrow checker. It's very hard to hack through stuff and get away with it, which makes it a particularly hard sell for game prototypes.
+
+* I know that it is a less common language to use so as a company I will have to have my staff learn Rust before they can start.
+
+* Rust could be tricky to hire for, and hiring someone who doesn't know Rust to work in Rust will come with a ramp-up period.
+
+### IDE/RLS story (28)
+
+Tooling around IDE is limited compared to Visual Studio with C++ or C#.´
```
Comment (17cupsofcoffee) [inline content/posts/survey-01/index.md]: There's a bullet point missing here and on lines 112/114.

I also kinda wonder if there's a way to format these bullets to make it a little clearer that they're quotes, but it's probably fine as is.

Comment (Wodann) [reply content/posts/survey-01/index.md]: I think marking it as quotes is actually a good point. The first time I read it, I wasn't sure whether they were a distilled, bullet-point summary of reflective quotes.

Comment (17cupsofcoffee) [reply content/posts/survey-01/index.md]: With the site's current styling, quotes would look like this:

image

If you do decide to go that route, please let me know if you're not happy with the fonts/italics/etc - I can submit a quick PR to change them before this post gets merged :)

**Comment by erlend-sh** [comment]
Changed all to quotes, except for the last lists which were chiefly distillations rather than handpicked quotes. If that's deemed confusing that can still be changed.

I'll merge this now so we can start spreading the word shortly. Formatting changes after publishing won't be a problem; the data won't change after all.

**Comment by 17cupsofcoffee** [comment]
Looks good :) Thank you for all your hard work on this!

**Pull request merged**